ImputationPipeline/imputation_interaction_python/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# default ubuntu version: 20.04
+ARG UBUNTU_VERSION=20.04
+FROM ubuntu:${UBUNTU_VERSION}
+
+# the following argument is needed for pkg-config to
+# install without hanging on user input
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
The Dockerfile compiles without these lines for me, do we need them?

Reply:
nope!